### ISSUE TITLE
Build fails on nodeJS 0.10 after upgrade to v1.0.3

### ISSUE THREAD

**Issue by jlchereau** [logentries/le_node#53]
Travis build fails with the following error on nodeJS 0.10 (not 0.11 or 0.12):

See https://travis-ci.org/kidoju/Kidoju-Blog/jobs/65060671
```
>> Mocha exploded!
>> ReferenceError: Symbol is not defined
>>     at Object.<anonymous> (/home/travis/build/kidoju/Kidoju-Blog/node_modules/le_node/lib/node_modules/logger.js:64:19)
>>     at Module._compile (module.js:456:26)
>>     at Object.Module._extensions..js (module.js:474:10)
>>     at Module.load (module.js:356:32)
>>     at Function.Module._load (module.js:312:12)
>>     at Module.require (module.js:364:17)
>>     at require (module.js:380:17)
>>     at Object.<anonymous> (/home/travis/build/kidoju/Kidoju-Blog/webapp/lib/logger.js:16:18)
>>     at Module._compile (module.js:456:26)
>>     at Object.Module._extensions..js (module.js:474:10)
```

Code reproduced below can be found at https://github.com/kidoju/Kidoju-Blog/blob/master/webapp/lib/logger.js. The line that breaks is line 16, that is ```LogEntries = require('le_node')```.
```
/* jshint node: true */

'use strict';

// More info at:
// https://github.com/logentries/le_node
// https://logentries.com/doc/nodejs/

var config = require('../config'),
    utils = require('./utils'),
    LogEntries = require('le_node'),
    logger = new LogEntries({ token: config.get('logentries:server:token')});

/**
 * Process entry.request if existing
 * @param entry
 * @returns {*}
 */
function process(entry) {
    if(entry.request) {
        var request = entry.request;
        entry = utils.deepExtend(entry, {
            agent: request.headers['user-agent'],
            ip: request.ip,
            query: request.query,
            sessionId: request.sessionId,
            url: request.url
        });
        delete entry.request;
    }
    return entry;
}

module.exports = {

    console: function(entry) {
        console.dir(entry, { showHidden: false, depth: 4, colors: true });
    },

    info: function(entry) {
        logger.log('info', process(entry));
    },

    warning: function(entry) {
        logger.log('warning', process(entry));
    },

    error: function(entry) {
        entry = process(entry);
        module.exports.console(entry);
        logger.log('err', entry);
    },

    critical: function(entry) {
        entry = process(entry);
        module.exports.console(entry);
        logger.log('crit', entry);
    },

    flush: function() {
        logger.closeConnection();
    }

};
```

**Comment by bathos** [comment]
Confirmed. I hadn't noticed because our 0.10 test had a false positive:

https://travis-ci.org/logentries/le_node/jobs/64932057

(I'll have to look into that too).

Fortunately the solution is very simple -- I just need to ensure it imports the missing natives (Symbol, but I think Promise, too) from CoreJS.

**Comment by bathos** [comment]
Fixed in 1.0.4.
